core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/molecule/ContentLoadingErrorView.kt

```diff
@@ -0,0 +1,88 @@
+package app.k9mail.core.ui.compose.designsystem.molecule
+
+import androidx.compose.animation.AnimatedContent
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import app.k9mail.core.ui.compose.designsystem.atom.text.TextSubtitle1
+import app.k9mail.core.ui.compose.theme.PreviewWithThemes
+
+@OptIn(ExperimentalAnimationApi::class)
+@Composable
+fun ContentLoadingErrorView(
+    state: ContentLoadingErrorState,
+    loading: @Composable () -> Unit,
+    error: @Composable () -> Unit,
+    modifier: Modifier = Modifier,
+    contentAlignment: Alignment = Alignment.Center,
+    content: @Composable () -> Unit,
```

Comment:
```suggestion
    state: ContentLoadingErrorState,
    modifier: Modifier = Modifier,
    contentAlignment: Alignment = Alignment.Center,
    loading: @Composable () -> Unit,
    error: @Composable () -> Unit,
    content: @Composable () -> Unit,
```

Reply:
The order of parameters is checked by Compose Detekt as outlined here: [Compose Detekt Ordering Rules](https://mrmans0n.github.io/compose-rules/rules/#ordering-composable-parameters-properly)

Wile the following version would be the most correct honoring: [Kotlin Default Arguments](https://kotlinlang.org/docs/functions.html#default-arguments)

```suggestion
    state: ContentLoadingErrorState,
    loading: @Composable () -> Unit,
    error: @Composable () -> Unit,
    content: @Composable () -> Unit,
    modifier: Modifier = Modifier,
    contentAlignment: Alignment = Alignment.Center,
```

An exception to this is made for the content lambda of a composable function, which is placed last. This allows for the utilization of Kotlin's trailing lambda syntax, making it easier to write complex layouts directly following the function call.